### ISSUE TITLE
Mixed Content When Deploying to HTTPS

### ISSUE THREAD

**Issue by dylan-mcdonald** [Smallmelo/food-accessibility-tutorial#2]
**Problem:** Examples don't load properly over HTTPS. On the JavaScript console, one gets:
> Mixed Content: The page at 'https://www.foo.com/food-accessibility-tutorial/Activity2/FinishedCode/' was loaded over HTTPS, but requested an insecure script 'http://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/leaflet.js'. This request has been blocked; the content must be served over HTTPS.

**Cause:** Script and link tags are hard wired to cleartext HTTP, e.g.:
```html
<link rel="stylesheet" href="http://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/leaflet.css" />
<script src="http://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/leaflet.js"></script>
```
**Solution:** Remove the hardwiring and use the agnostic // instead:
```html
<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/leaflet.css" />
<script src="//cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/leaflet.js"></script>
```

**Remark**: OpenStreetMap still loads the icons using HTTP when they're plotted on the map. HTTPS users will get a lot of mixed content warnings, but it will still work. :)

**Comment by meitar** [comment]
FWIW, I fixed this in fixed in PR #3. The CDNs support HTTPS, so hardcoding that is even better than server-relative `//` URLs.

**Comment by dylan-mcdonald** [comment]
:+1: True enough. All my servers use HTTPS, so I'm used to server relative URLs :smile_cat:
